test/unit/specs/csv.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { TransactionStatuses, TransactionTypes } from '@brafdlog/israeli-bank-scrapers-core/lib/transactions';
+import { EnrichedTransaction } from '../../../src/originalBudgetTrackingApp/commonTypes';
+import { serializeTransactions, parseTransactions } from '../../../src/originalBudgetTrackingApp/export/outputVendors/csv/csv';
```

Comment:
```suggestion
import { EnrichedTransaction } from '@/originalBudgetTrackingApp/commonTypes';
import { serializeTransactions, parseTransactions } from '@/originalBudgetTrackingApp/export/outputVendors/csv/csv';
```

Reply:
Ok.
BTW for some reason VS code IDE doesn't recognize the file when I follow this format in the unit tests (though it works for the main code). The tests compile and run fine, but it's a bit of a pain working from the IDE.
Note that the other unit tests also follow that long format, I don't know if that's the reason.